### DISCUSSION TITLE
Insensitive sorting in react table v7

### DISCUSSION THREAD

**Discussion by Kavimugilan17** [TanStack/table#2167] [General] [answered]
![image](https://user-images.githubusercontent.com/58725968/79713883-b348b200-82ec-11ea-922a-940d3090b87f.png)

After sorting Lower cases are went down.

Link: https://codesandbox.io/s/holy-voice-3zeqv?file=/src/App.js

**Comment by Kavimugilan17** [comment, marked as answer]
Below Code is working fine.

Custom Sorting Code:
-----------------------------
const sortItems = (prev: Row<ITaskMetadata>, curr: Row<ITaskMetadata>, columnId: string) => {
  if (prev.original[columnId].toLowerCase() > curr.original[columnId].toLowerCase()) {
    return 1;
  } else if (prev.original[columnId].toLowerCase() < curr.original[columnId].toLowerCase()) {
    return -1;
  } else {
    return 0;
  }
};

Column:
-----------------
   {
    Header: 'Sample Header',
    accessor: 'sampleHeader',
    sortType: (prev, curr, columnId) => {
      return sortItems(prev, curr, columnId);
    }
  }

**Comment by smcalilly** [comment]
this doesn't quite work but it's on the right track. if you click the sort button three times in a row, it goes back to the column's original sort that happened on page load. i'm trying to figure out an issue, but might override this by sorting the data once before the table component mounts

**Reply by as-zlynn-philipps** [reply]
is that not expected behavior?
